src/api/config/routes.rb

```diff
@@ -16,8 +16,11 @@ def self.matches?(request)
 class APIMatcher
   def self.matches?(request)
     format = request.format.to_sym || :xml
-    # Rails.logger.debug "MATCHES #{format}"
-    format == :xml || format == :json
+    format == :xml || format == :json || public_or_about_path?(request)
+  end
+
+  def self.public_or_about_path?(request)
+    /^\/public|^\/about/  =~ request.fullpath
```

Comment:
JFYI, this should also work:
/^\/(?:public|about)/  =~ request.fullpath
or
request.fullpath.start_with?("/public", "/about")